public/js/app/components/statistics/MemosSummary.jsx

```diff
@@ -93,6 +108,7 @@ function MemosNumbersCharts({ statisticsResult }) {
   const { combinedMemosPerSchool: docsPerSchool, periods, year } = statisticsResult
   const { schools = {} } = docsPerSchool
   const [{ languageIndex }] = useWebContext()
+  schools['All schools'] = addAllSchoolsData(docsPerSchool)
```

Comment:
`All schools`  better to be replaced with `allSchools`

Reply:
done